src/components/Modal.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+const Modal = ({ show, modalTitle, description, close }) => {
+  return (
```

Comment:
simplify the show integration. And instead of adding the complete R-DOM in a conditional statement, add a one line code before the Line#4
```
if(!show) return <></>
```

and remove the condition in existing return

Reply:
done